### PR TITLE
Bellasbranch

### PR DESCRIPTION
**Pull request by isabessa05**
# Changelog
All notable changes to this project will be documented in this file.

# Issue(s)
Include issue here as so:
#24 

## General Description:
Addition of Chakra UI to the project. 

### Added:
Chakra UI was installed in the project and the router was added to the main App.js component.
ExamDetail page was also created, but not called into App.js yet.

### Changed:
The only change made was the addition of the router for Chakra UI into App.js, wrapping it's components.

### Removed
Nothing was removed.

### Planned
The next step would be work through the ExamDetail page and add the main jsx. 

### Required Reviewers
Please add required reviewers here like:
@ylopez25 @ccelest1

### CODE REVIEW

File: client/package.json [hunk 1]
```diff
@@ -3,9 +3,13 @@
   "version": "0.1.0",
   "private": true,
   "dependencies": {
+    "@chakra-ui/react": "^2.8.2",
```
Comment (ccelest1) [inline client/package.json:6]: Nice, yes we planning on going with chakra, emotion, framer motion, with react

File: client/package.json [hunk 2]
```diff
@@ -3,9 +3,13 @@
...
+    "@emotion/react": "^11.11.3",
+    "@emotion/styled": "^11.11.0",
     "@testing-library/jest-dom": "^5.16.5",
     "@testing-library/react": "^13.4.0",
     "@testing-library/user-event": "^13.5.0",
+    "framer-motion": "^11.0.3",
     "react": "^18.2.0",
```
Comment (ccelest1) [inline client/package.json:13]: I see you do not have react-router-dom here, please install that

File: client/src/App.js [hunk 3]
```diff
@@ -1,18 +1,14 @@
...
 
   return (
-    <div className="App">
-      <header className="App-header">
-        <p>
-          {response}
-        </p>
-      </header>
-    </div>
+    <ChakraProvider>
```
Comment (ccelest1) [inline client/src/App.js:9]: While this is great to be provided, this isn't enough for a PR just yet
Please add the respective pages pertaining to issues:
#24
#26
and then a PR can be requested for merging

File: client/src/ExamDetails.js [hunk 4]
```diff
@@ -0,0 +1,13 @@
+function ExamDetails() {
+
+    return (
+        <>
+        <div>
+            <img src="https://medlineplus.gov/images/Xray_share.jpg"/>
```
Comment (ccelest1) [inline client/src/ExamDetails.js:6]: Is this for visual fe testing?

**Review by ccelest1 — CHANGES_REQUESTED** [review]
I don't know if this rises to PR level.
I left other comments for consideration.

File: client/src/ExamDetails.js (hunk 4 above)
Comment (isabessa05) [reply client/src/ExamDetails.js:6]: Yes, so I can use it to adjust the card and image before calling the one exam.

**Pull request closed without merging**